tests/integration/callbacks/test_clientside.py

```diff
@@ -0,0 +1,72 @@
+from selenium.webdriver.support.select import Select
+import time
+
+app = """
+library(dash)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+
+app <- Dash$new()
+
+app$layout(htmlDiv(list(
+    dccInput(id='input'),
+    htmlDiv(id='output-clientside'),
+    htmlDiv(id='output-serverside')
+    )
+    )
+)
+
+app$callback( 
+  output(id = "output-serverside", property = "children"),
+  params = list(
+    input(id = "input", property = "value")
+  ),
+  function(value) {
+    sprintf("Server says %s", value)
+  }
+)
+
+app$clientside_callback(
+  output('output-clientside', 'children'),
+  params=list(input('input', 'value')),
+  clientsideFunction(
+    namespace = 'clientside',
+    function_name = 'display'
+    )
+)
```

Comment:
What if we allow `app$callback` to accept either an R `function` or a `clientsideFunction`, determine which it is based on type, and do away with `app$clientside_callback`? It's a little different from Python, but that's just because of Python's decorators vs R's anonymous functions. It's still plenty clear, I think, as you've still got an explicit call to `clientsideFunction`.

Reply:
I wondered about that as I was working on this PR; using the same syntax for both seems perfectly fine to me. I'll go ahead and make this change.

Reply:
fixed in e4fffaf